README.md

```diff
@@ -0,0 +1,48 @@
+# NYPL Header App
+The NYPL Header App can be used for projects across the NYPL organization to import the Header and Footer components.
+
+# Running the app locally
+Install all dependencies
+```sh
+$ npm install
+```
+Builds the app into `./dist` and preview the app at `localhost:4173`
+```sh
+$ npm run prod
+```
+
+## Running the locally app with Docker
+Build the Docker image
```

Comment:
Add a line that the developer will need Docker running on their machine in order to build the image and run the app through Docker.